src/tools/illink/test/Mono.Linker.Tests.Cases/RequiresCapability/RequiresOnAttributeCtor.cs

```diff
@@ -9,6 +9,7 @@
 
 namespace Mono.Linker.Tests.Cases.RequiresCapability
 {
+	[IgnoreTestCase ("This test is still not enabled by NativeAOT", IgnoredBy = ProducedBy.NativeAot)]
```

Comment:
As mentioned offline, I think these messages should be a link to a bug that describes the work necessary to categorize these issues.